nautobot/docs/user-guide/administration/upgrading/from-v1/tables/v2-code-removals.yaml

```diff
@@ -47,6 +47,8 @@
   Class/Function(s): "PluginMenuButton"
 - Module: "nautobot.extras.plugins"
   Class/Function(s): "PluginMenuItem"
+- Module: "nautobot.extras.urls"
+  Class/Function(s): "jobs/results/<uuid:pk>/"
```

Comment:
This isn't a class/function, I'm not sure it's appropriate in this table. I think documenting it in the file below is probably preferable/sufficient.